tests/conftest.py

```diff
@@ -3,52 +3,66 @@
 # conftest.py
 #
 
-from __future__ import print_function, division, absolute_import, unicode_literals
+from __future__ import absolute_import, division, print_function, unicode_literals
 
-import re
-import pytest
 import importlib
 import inspect
-from .sqladbs import prepare_testdb as sqla_prepdb
+import re
+
+import pytest
+from pytest_postgresql.janitor import DatabaseJanitor
+
 from .pwdbs import prepare_testdb as pw_prepdb
-from pytest_postgresql.factories import DatabaseJanitor
+from .sqladbs import prepare_testdb as sqla_prepdb
 
 
 def pytest_addoption(parser):
-    """ Add new options to the pytest command-line """
+    """Add new options to the pytest command-line"""
     # only run peewee tests
-    parser.addoption('--peewee', action='store_true', default=False,
-                     help='Only run tests for peewee dbs')
+    parser.addoption(
+        "--peewee",
+        action="store_true",
+        default=False,
+        help="Only run tests for peewee dbs",
+    )
 
```

Comment:
I'll just comment here but no need to change it.  This is my other grievance with formatters.  Not a fan at all when they do this.

Reply:
Yeah, that's a problem right now. We have a linter but not a formatter. My default VSCode config will apply `ruff format` on save, which maybe shouldn't for this repo, or should do it only on files that one wants to keep formatted. Something to discuss for 1.x.